spec/support/helpers/general_spec_helpers.rb

```diff
@@ -50,4 +62,13 @@ def with_env(vars)
   ensure
     vars.each_pair { |k, _| ENV.delete(k) }
   end
+
+  private
+
+  def fixture(dir, name)
+    path = Rails.root.join("spec/fixtures/#{dir}/#{name}")
+
+    # attaching a media file does not require opening it
+    dir == "media/audio" ? path : File.open(path)
```

Comment:
Can you see the parallel here with the "knowledge" thing we discussed earlier? `fixture` is a generic method. It shouldn't "know" about specifics like the `"media/audio"` path. 

I understand you don't want to open the file in that case. Can you think of other ways of solving that problem? Feel free to Slack me some ideas instead of doing a whole new PR>